### ISSUE TITLE
Add labels to Docker image

### ISSUE THREAD

**Issue by TobiX** [axllent/mailpit#267]
Hi!

First: Thanks for this tool. I have switched from MailHog und really like all the new features (tags, dark theme, etc.)

I currently have just a small request: Could you add some [labels](https://github.com/opencontainers/image-spec/blob/main/annotations.md#back-compatibility-with-label-schema) to the Docker image?

I think a fairly uncontroversial set would be:
```dockerfile
LABEL org.opencontainers.image.source="https://github.com/axllent/mailpit" \
  org.opencontainers.image.url="https://mailpit.axllent.org/" \
  org.opencontainers.image.licenses="MIT"
```

This helps tools like [Renovate](https://github.com/renovatebot/renovate) (and Dependabot?) to link and embed changelogs into update PRs (for example when using docker images in Helm charts)

**Comment by axllent** [comment]
Thanks for this, I'll look into it and add something in the near future 👍

**Comment by axllent** [comment]
I've added the following to the Dockerfile:
```
LABEL org.opencontainers.image.title="Mailpit" \
  org.opencontainers.image.description="An email and SMTP testing tool with API for developers" \
  org.opencontainers.image.source="https://github.com/axllent/mailpit" \
  org.opencontainers.image.url="https://mailpit.axllent.org" \
  org.opencontainers.image.documentation="https://mailpit.axllent.org/docs/" \
  org.opencontainers.image.licenses="MIT"
```
I trust this will be sufficient?

Unless you have good reason to require this "now", I'll wait until I have a few more changes before releasing a new version :+1:

**Comment by TobiX** [comment]
Thanks! I think that's reasonable.

**Comment by axllent** [comment]
This has finally been released in v1.15.1. I :+1: 

Please confirm it works as you expect?

**Comment by TobiX** [comment]
Sorry for the late reply, can confirm it works now, see https://github.com/TobiX/test-renovate-stuff/pull/7
